src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -1215,6 +1215,15 @@ TIP: Since it runs a graphical application, the task will never exit and run for
 you want to check manually some results via the graphical application and then exit the application.
 The task will then terminate normally.
 
+=== Troubleshoot a Workflow Execution
+
+When a task has *IN_ERROR* status, a right click on it allows you to select _Mark as finished and resume_. 
+This will set the task status as *FINISHED* and then resume job execution. 
+This allows an operator to manually perform actions to the *IN_ERROR* task, without cancelling the whole job execution.
+
```

Comment:
You could make more clear that a task which is finished is not executed again:

A task which is finished with _Mark as finished and resume_ is not executed again,  subsequent tasks will execute as if the task executed correctly. 

Furthermore we can explain how the variables are propagated. Do we lose all variables (we need to check that). Ideally we should forward the variables but not add variables to it (because the task couldn't produce variables).